src/main.ts

```diff
@@ -8,9 +8,18 @@ import {ExecOptions} from '@actions/exec/lib/interfaces'
 const IS_WINDOWS = process.platform === 'win32'
 const VS_VERSION = core.getInput('vs-version') || 'latest'
 const VSWHERE_PATH = core.getInput('vswhere-path')
+const ALLOW_PRERELEASE = core.getInput('vs-prerelease') || 'false'
 
 // if a specific version of VS is requested
-let VSWHERE_EXEC = '-products * -requires Microsoft.Component.MSBuild -property installationPath -latest '
+let VSWHERE_EXEC = '-products * -requires Microsoft.Component.MSBuild -property installationPath '
```

Comment:
`-latest` and `-prerelease` are not mutually exclusive. `-latest` just picks the latest version (or last update, if versions are the same for the latest) regardless of release or prerelease status. `-prerelease` includes prereleases into that mix.

Reply:
Sounds good. I originally had it that way but was unsure. Fixed!